go/consensus/tendermint/apps/staking/state/state.go

```diff
@@ -703,12 +703,14 @@ func (s *MutableState) TransferFromCommon(
 	ctx *abciAPI.Context,
 	toAddr staking.Address,
 	amount *quantity.Quantity,
+	escrow bool,
```

Comment:
Maybe update documentation above with the added field and that it takes into account commission schedule.

Reply:
Good idea, updated.